src/frontend/cli.rs

```diff
@@ -79,6 +76,9 @@ enum Contract {
     /// Replicate a third-party contract into this project
     #[clap(name = "fork", bin_name = "fork")]
     ForkContract(ForkContract),
+    /// Publish contracts on chain
+    #[clap(name = "publish", bin_name = "publish")]
+    Publish(Publish),
```

Comment:
Good point. I changed this in this PR because I have an incoming collision with `clarinet lambdas publish` (or so) that I'd like to avoid. I'd be ok with keeping `check` and `test` as a root commands for now (would be checking/testing contracts+lambdas), but if you have a strong opinion on this one, I can also move these features as `contracts` subcommands.

Reply:
Ah, ok. That makes sense. If `check` and `test` will be covering more than just the contracts, then I think keeping them at the top level makes sense.